customer-commons/src/test/java/no/unit/nva/customer/model/CustomerDtoTest.java

```diff
@@ -27,6 +27,29 @@
 
 class CustomerDtoTest {
 
+    @Test
+    void doiAgentMergeTest() {
+        var doiAgent = randomCustomer()
```

Comment:
Call the variable "customer"

Reply:
That was dead code... (deleted)

Reply:
` var doiAgent = randomCustomer().getDoiAgent()
                           .addPassword(randomString());
`